keystone/api/users.py

```diff
@@ -187,13 +186,42 @@ def _get_user(self, user_id):
 
         GET/HEAD /v3/users/{user_id}
         """
-        ENFORCER.enforce_call(
-            action='identity:get_user',
-            build_target=_build_user_target_enforcement
-        )
+        # As a particular override for Chameleon, we allow users to get
+        # other users in their same project.
+        if self._in_same_project_as_user(user_id):
+            ks_flask.common.set_unenforced_ok()
```

Comment:
Why exactly is this required? It seems to me you could just skip the ENFORCER call in the else branch.

Reply:
All API calls require a call to enforcement, or explicitly stating that it's unenforced. Without the this call, it throws.